hydroflow_lang/src/graph/flat_graph_builder.rs

```diff
@@ -290,12 +290,12 @@ impl FlatGraphBuilder {
 
         let mut node_mapping = BTreeMap::new();
```

Comment:
The description I wrote of merge_in is not very helpful, I think we should point out that the Ends it returns are the input and output of the module that has been imported into `self`.

I also think that `merge_in` is kind of a misleading name, it's not really doing any 'merging'. I think a better name might be 'import_flatgraph', since what it returns is a handle to the imported flatgraph, and that handle can be used to complete the stitching.

Not sure about the second idea, but the first idea I think would be helpful.